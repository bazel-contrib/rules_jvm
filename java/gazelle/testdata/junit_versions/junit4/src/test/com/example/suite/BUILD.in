# gazelle:java_test_mode suite
