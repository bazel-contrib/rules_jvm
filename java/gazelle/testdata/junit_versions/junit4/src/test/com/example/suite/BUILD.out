load("@contrib_rules_jvm//java:defs.bzl", "java_test_suite")

# gazelle:java_test_mode suite

java_test_suite(
    name = "suite",
    srcs = ["ExampleTest.java"],
    deps = ["@maven//:junit_junit"],
)
